frontend/app/features/events/views/event.html

```diff
@@ -1,31 +1,38 @@
 <div class="container event-page" ng-hide="eventNotFound">
     <div class="row">
         <h1>{{event.title}} <span class="date pull-right">{{event.date | date : "dd.MM.yyyy"}}</span></h1>
-        <a href="/events/{{event.id}}/edit" class="btn btn-primary pull-right">Edit event</a>
+        <a href="/events/{{event.id}}/edit" ng-if="currentUser.role === 'admin'" class="btn btn-primary pull-right">Edit event</a>
     </div>
-    <div class="row" >
-      <div class="col-md-6">
-      <div id="map" map ></div>
-      </div>
-      <div class="col-md-6">
-        <p>{{event.desc}}</p>
-        <div class="btn-group" ng-if = "!isSubscribe">
-         <button type="button" class="btn btn-success btn-lg" ng-click="subscribe()" ng-model="Subscribe">Subscribe</button>
-       </div>
-       <div  ng-if = "isSubscribe">
-         <div class="alert alert-success" >
-          You have successfully subscribed to event
-         </div>
-         <button type="button" class="btn " ng-click="unSubscribe()" >Unsubscribe</button>
-
-       </div>
+    <div class="row">
+        <div class="col-md-5">
+            <div class="row">
+                <div id="map" map></div>
+            </div>
+        </div>
+        <div class="col-md-6 col-md-offset-1">
 
-      </div>
-  </div>
-</div>
-<div class="container event-page" ng-show="eventNotFound">
-    <div class="alert alert-danger">
-        <h2>404</h2>
-        <p>Requested event not a found</p>
+            <div class="row">
+                <p>{{event.desc}}</p>
+                <div class="flash-block">
+                    <div ng-class="{ 'alert': flash, 'alert-success': flash.type === 'success', 'alert-danger': flash.type === 'error' }" ng-if="flash" ng-bind="flash.message"></div>
+                </div>
+                <div class="btn-group" ng-if="UserId">
+                    <button ng-cloak type="button" ng-class="isSubscribe ? '' : 'btn-success'" class="btn btn-lg" ng-click="goEvent()">{{SubscribeMessage}}</button>
+                </div>
+                <div class="row" ng-show="userList.length > 0">
+                  <hr/>
+                    <h4 ng-hide="userList[0].id == UserId">Will take part:</h4>
+                    <h4 ng-show="userList[0].id == UserId">You are first, who will take part in event</h4>
+                    <ul ng-repeat="user in userList">
+                        <li ng-show="user.id != UserId">{{user.full_name}}</li>
+                    </ul>
```

Comment:
it would better to create one list for all users. For example,
[ul]
[li]user[/li]
[li]user[/li]
[li]user[/li]
[/ul]